README.md

```
@@ -50,3 +50,36 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/app/bui
+`npx wrangler d1 execute production --local --command="SELECT * FROM users"`
+
+For local development add SSL certificate(you will get issue in the browser regarding insecure connection):
+`sudo security add-trusted-cert -d -r trustRoot -k "/Library/Keychains/System.keychain" certificates/localhost.pem`
+Certificate should be generated after in `./certificates` directory after first run on `npm run dev` - wrangler dev with https flag
+You might need to close the browser application and reopen to refresh ssl certificates.
+
+##CF R2
+`wrangler r2 bucket create your-bucket-name`
+
+Connet bucke to a custom domain that is already managed by cloudflare:
+https://developers.cloudflare.com/r2/buckets/public-buckets/#connect-a-bucket-to-a-custom-domain
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)